packages/remix-dev/cli/run.ts

```diff
@@ -278,6 +296,8 @@ export async function run(argv: string[] = process.argv.slice(2)) {
 
       await commands.create({
         appTemplate: flags.template ?? answers.appTemplate,
+        templateType:
+          templateType! || (answers.appType === "stack" ? "repo" : "template"),
         projectDir,
         remixVersion: flags.remixVersion,
         installDeps: flags.install ?? answers.install,
```

Comment:
can you change these to logical ORs and I'll close #2440?
using nullish coalescing prevents our CLI from showing the minimum node version error
```suggestion
        installDeps: flags.install || answers.install,
        useTypeScript: flags.typescript || answers.useTypeScript,
```